src/www/pages/patterns/contact-details/index.njk

```diff
@@ -65,15 +64,15 @@
 
     {{
         compExample([
-            wmndsInsetText({html:
+            wmndsInsetText({
+                contentHTML:
                 wmndsContactDetails({
                     warningText: "We are currently experiencing problems with our <br>telephone systems and cannot answer calls",
                     warningIcon: "general-warning-triangle",
-                    content: "
-    <a href=\"mailto:customerservice@tfwm.org.uk\">customerservice@tfwm.org.uk</a><br>
-    Telephone: <a href=\"tel:03453036760\">0345 303 6760</a><br>
-    Monday to Tuesday and Thursday to Friday, 9am - 5pm, <br>
-    Wednesday, 9.30am - 5pm"
+                    content: "<a href=\"mailto:customerservice@tfwm.org.uk\">customerservice@tfwm.org.uk</a><br>
+                    Telephone: <a href=\"tel:03453036760\">0345 303 6760</a><br>
+                    Monday to Tuesday and Thursday to Friday, 9am - 5pm, <br>
+                    Wednesday, 9.30am - 5pm"
                 }) | indent(6) | trim
```

Comment:
There's a line break missing here (note the gap above the email address):

LIVE SITE:
![image](https://user-images.githubusercontent.com/13015201/102883460-f775ee80-4447-11eb-9aad-317a623fc1c1.png)


With your changes:
![image](https://user-images.githubusercontent.com/13015201/102883481-00ff5680-4448-11eb-9ce7-da88a65cfe33.png)